lib/hello_sign/api/signature_request.rb

```diff
@@ -657,6 +657,17 @@ def update_signature_request(opts)
         path = "/signature_request/update/#{signature_request_id}"
         HelloSign::Resource::SignatureRequest.new post(path, body: opts)
       end
+
+      # Releases a held SignatureRequest that was claimed and prepared from an UnclaimedDraft.
+      # @option opts [String] signature_request_id The ID of the SignatureRequest to release.
+      #
+      # @return [HelloSign::Resource::SignatureRequest] a SignatureRequest
+      #
+      # @example
+      #   @client.release_signature_request signature_request_id: '75cdf7dc8b323d43b347e4a3614d1f822bd09491'
+      def release_hold_signature_request(opts)
```

Comment:
can you rename this to match the example code in the line above or vice versa? i'd recommend using `release_signature_request` or `release_on_hold_signature_request` to avoid confusion